src/components/navbar.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { BiUser } from 'react-icons/bi';
+import { NavLink } from 'react-router-dom';
+import classes from './navbar.module.css';
+
+const SiteNavbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      Text: 'BOOKS',
+    },
+    {
+      id: 2,
+      path: '/categories',
+      Text: 'CATEGORIES',
+    },
+  ];
+  return (
+    <>
+      <nav className={classes.nav__bar}>
+        <h1 className={classes.bookstore}>Bookstore CMS</h1>
+        <ul>
+          {links.map((nlink) => (
+            // eslint-disable-next-line
+            <li key={nlink.id} className={'li' + nlink.id.toString()}>
+              <NavLink to={nlink.path} className={classes.navlinks}>
+                {nlink.Text}
+              </NavLink>
+            </li>
+          ))}
+        </ul>
+        <h1 className={classes.user}>
+          <BiUser />
+        </h1>
+      </nav>
```

Comment:
- [ ] Good job so far 👏 Please add some space on the right side of the nav bar as well as on the bookList component as on the design given 

Your design
![image](https://user-images.githubusercontent.com/90222110/225726004-565f1c83-4b37-4b9a-b08f-188c6a3719ee.png)

Design given
![image](https://user-images.githubusercontent.com/90222110/225726370-27655bf0-e52d-4ff4-a645-4321bc538e07.png)